bio/bowtie2/align/environment.yaml

```
@@ -3,7 +3,7 @@ channels:
   - bioconda
   - nodefaults
 dependencies:
-  - bowtie2 =2.5.4
-  - picard-slim =3.3.0
+  - bowtie2 =2.5
```

For versions control
```suggestion
  - bowtie2 =2.5.4
```